configure.php

```diff
@@ -60,6 +60,27 @@ function replace_in_file(string $file, array $replacements): void {
     );
 }
 
+function remove_readme_paragraphs(string $file): void {
```

Comment:
Use camelCase, just like the other functions.

Reply:
I did copy from here, but i will change it
https://github.com/spatie/package-skeleton-php/blob/ea4bc716df055b36cb8b19236fc042739d710b3b/configure.php#L50-L55

Reply:
Fixed